internal/cmd/commands/authenticate/oidc.go

```diff
@@ -72,12 +72,6 @@ func (c *OidcCommand) Run(args []string) int {
 		return base.CommandUserError
 	}
 
-	switch {
-	case c.FlagAuthMethodId == "":
-		c.PrintCliError(errors.New("Auth method ID must be provided via -auth-method-id"))
-		return base.CommandUserError
-	}
-
```

Comment:
Do we not still need this for OIDC?

Reply:
I was waiting on the first pass on password auth method before doing OIDC. If you think the pattern in password is ok, I can get that same implemented in OIDC next.